src/components/common/SimpleTable/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { PropsWithChildren } from 'react'
+import styled from 'styled-components'
+
+const Wrapper = styled.table<{ $numColumns?: number }>`
+  font-size: var(--font-size-default);
+  background: var(--color-primary);
+
+  margin: 0;
+  height: calc(100% - 8.8rem);
+  padding: 0;
+  box-sizing: border-box;
+  width: 100%;
+  border-spacing: 0;
+  display: inline-grid;
+  grid-template-areas:
+    'head-fixed'
+    'body-scrollable';
+
+  tr {
+    text-align: left;
+    padding: 0;
+  }
+
+  tr > td {
+    padding: 0;
+    color: var(--color-text-secondary2);
+    transition: color 0.1s ease-in-out;
+    box-sizing: border-box;
+  }
+
+  > thead {
+    grid-area: head-fixed;
+    position: sticky;
+    top: 0;
+    height: auto;
+    display: flex;
+    align-items: center;
+  }
+
+  > thead > tr {
+    color: var(--color-text-secondary2);
+    display: grid;
+    width: calc(100% - 0.6rem);
+  }
+
+  > thead > tr > th {
+    font-weight: var(--font-weight-normal);
+    &:not(:first-of-type) {
+      text-align: right;
+    }
+  }
+
+  > tbody {
+    grid-area: body-scrollable;
+    overflow-y: auto;
+    overflow-x: hidden;
+    height: 100%;
+    box-sizing: border-box;
+    padding: 0;
+  }
+
+  > tbody > tr {
+    display: grid;
+    width: 100%;
+    transition: background 0.1s ease-in-out;
+    border-bottom: 0.1rem solid var(--color-border);
+    &:hover {
+      background: var(--color-text-hover);
+      > td {
+        color: var(--color-text-primary);
+      }
+    }
+
+    &:last-of-type {
+      margin: 0 0 5rem;
+    }
+  }
+
+  > thead > tr > th,
+  > tbody > tr > td {
+    height: 3rem;
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+
+    &:first-of-type {
+      padding-left: 1rem;
+      justify-content: flex-start;
+    }
+  }
+
+  > thead > tr,
+  > tbody > tr {
+    align-items: center;
+    ${({ $numColumns }): string => ($numColumns ? `grid-template-columns: repeat(${$numColumns}, 1fr)` : '')}
```

Comment:
Maybe the `;` is missing?
```suggestion
    ${({ $numColumns }): string => ($numColumns ? `grid-template-columns: repeat(${$numColumns}, 1fr);` : '')}
```

Reply:
```suggestion
    ${({ $numColumns }): string | false => $numColumns && `grid-template-columns: repeat(${$numColumns}, 1fr);`}
```

Reply:
Actually, I'll revert David's suggestion because of types and just use Anxo's `;` fix.
![screenshot_2021-01-20_10-10-38](https://user-images.githubusercontent.com/43217/105216716-c7ce0b00-5b07-11eb-8113-8efae7e625d1.png)